spec/feature/authentication_spec.rb

```diff
@@ -0,0 +1,52 @@
+feature 'authentication' do
```

Comment:
Easy to understand, clear structure for feature tests (arrange, act and assert).